resource-managers/yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnClusterSuite.scala

```diff
@@ -205,15 +205,15 @@ class YarnClusterSuite extends BaseYarnClusterSuite {
       .startApplication()
 
     try {
-      eventually(timeout(30.seconds), interval(100.milliseconds)) {
+      eventually(timeout(180.seconds), interval(100.milliseconds)) {
```

Comment:
You can write "3.minutes" in cases like this and others below, but it doesn't matter.